contracts/authorization/src/contract.rs

```diff
@@ -614,30 +610,50 @@ fn retry_msgs(
                     priority: authorization.priority,
                 }),
             )?;
-
+            // Update the state
+            callback_info.execution_result = ExecutionResult::InProcess;
             // Create the callback again
             let callback_request = CallbackRequest {
                 receiver: env.contract.address.to_string(),
                 // We will use the ID to know which callback we are getting
                 msg: to_json_binary(&PolytoneCallbackMsg::ExecutionID(execution_id))?,
             };
-
-            create_wasm_msg_for_processor_or_bridge(
+            messages.push(create_msg_for_processor_or_bridge(
                 deps.storage,
                 execute_msg_binary,
                 &callback_info.domain,
                 Some(callback_request),
-            )?
+            )?);
         }
         _ => {
```

Comment:
why `_`? it can only be None, being ambiguous here has to benefit and can actually be worse

Reply:
It can not only be None, that's why I used _.
First case is `Some(ttl) if !ttl.is_expired(&env.block)`
It can be None or Some and be expired. For both possible cases I want the same action

Reply:
Right, forgot this match syntax